packages/@react-spectrum/menu-trigger/stories/Menu.stories.tsx

```diff
@@ -144,28 +144,28 @@ storiesOf('Menu', module)
     () => (
       <Popover isOpen hideArrow>
         <Menu autoFocus={false} onSelectionChange={action('onSelectionChange')} defaultSelectedKeys={['2']}>
-          <Section key="section1" title="Section 1">
-            <Item key="1">
+          <Section title="Section 1">
+            <Item>
               One
             </Item>
-            <Item key="2">
+            <Item staticKey="2">
```

Comment:
Need to use a different prop than `key` for static collections since the `key` is mutated by `React.Children.toArray` and `getKey` in CollectionBuilder, resulting in a mismatch between the user provided `defaultSelectedKeys` array and what the collection sees in its `keyMap`. `staticKey` is intended to only be used for static collection, open to name suggestions

Reply:
is there a reason we are not doing this like non-static collection with itemKey prop? and allowing user to chose a prop name

        <Menu itemKey="name" >
          <Item name="one" >One</Item>
          <Item name="two" >Two</Item>
          <Item name="tree">Three</Item>
        </Menu>

Reply:
Hm, that is a interesting alternate route to take. I was initially going for a default key prop that the user could supply, but this might be more flexible. As of now static collections don't support this because their nodes aren't supplied with a proper value that `getKey` can use but I can see if I can remedy this.

Reply:
Looks like I can get this to work, just requires some minor tweaking in CollectionBuilder. I'll bring this up in my design review meeting today and see what we'll settle on

Reply:
@pst67662 as per design review meeting, we'll go with the current implementation